tests/unit_tests/test_wave_01.py

```diff
@@ -49,7 +49,13 @@ def test_removing_not_found_is_false():
 
     result = vendor.remove(item)
 
-    raise Exception("Complete this test according to comments below.")
+    # raise Exception("Complete this test according to comments below.")
     # *********************************************************************
     # ****** Complete Assert Portion of this test **********
     # *********************************************************************
+    assert result == False
+    assert vendor.inventory == ["a", "b", "c"]
+    assert vendor.inventory[0] == "a"
+    assert vendor.inventory[1] == "b"
+    assert vendor.inventory[2] == "c"
+    assert len(vendor.inventory) == 3
```

Comment:
The last 4 asserts are not needed here. If the assert on line 57 is true, then the remaining ones are guaranteed to be true as well.